src/test/java/LoginPageTest.java

```diff
@@ -0,0 +1,67 @@
+import org.assertj.core.api.Assertions;
+import org.assertj.core.api.SoftAssertions;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import pages.LoginPage;
+
+import java.time.Duration;
+
+public class LoginPageTest {
+    private WebDriver driver;
+
+    @BeforeClass
+    public void setUp() {
+        System.setProperty("webdriver.chrome.driver", "src/main/resources/drivers/chromedriver.exe");
+        driver = new ChromeDriver();
+        driver.manage().window().maximize();
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(5));
+        driver.get("https://magento.softwaretestingboard.com/customer/account/login/");
+    }
+
+    @Test
+    public void loginPage_PageTitle_isPageTitleCorrectlyDisplayed() {
```

Comment:
To keep consistency with the rest test method names `isPageTitleCorrectlyDisplayed` should be replaced with `**Has**PageTitleCorrectlyDisplayed`